assembly_and_binning/tutorial_assembly_and_binning.md

```diff
@@ -0,0 +1,634 @@
+
+```
+[Fri Jul 22 09:01:39 2022]
+Error in rule run_checkm_lineage_wf:
+    jobid: 34
+    output: SRR8859675/binning/DASTool/checkm/completeness.tsv, SRR8859675/binning/DASTool/checkm/storage/tree/concatenated.fasta
+    log: SRR8859675/logs/binning/DASTool/checkm.log (check log file(s) for error message)
+    conda-env: /home/stamps2022/atlas_workflow/databases/conda_envs/a489c28d87eeb559bd025b7924d669bc
+    shell:
+
+        rm -r SRR8859675/binning/DASTool/checkm
+        checkm lineage_wf 
+            --file SRR8859675/binning/DASTool/checkm/completeness.tsv            
+            --tmpdir /tmp             
+            --tab_table             
+            --quiet            
+            --extension fasta             
+            --threads 8             
+            SRR8859675/binning/DASTool/bins             
+            SRR8859675/binning/DASTool/checkm &> SRR8859675/logs/binning/DASTool/checkm.log
+
+        (one of the commands exited with non-zero exit code; note that snakemake uses bash strict mode!)
+
+Removing output files of failed job run_checkm_lineage_wf since they might be corrupted:
+SRR8859675/binning/DASTool/checkm/storage/tree/concatenated.fasta
+Shutting down, this might take some time.
+Exiting because a job execution failed. Look above for error message
+Note the path to the log file for debugging.
+Documentation is available at: https://metagenome-atlas.readthedocs.io
+Issues can be raised at: https://github.com/metagenome-atlas/atlas/issues
+The code used to generate one or several output files has changed:
+    To inspect which output files have changes, run 'snakemake --list-code-changes'.
+    To trigger a re-run, use 'snakemake -R $(snakemake --list-code-changes)'.
+The input used to generate one or several output files has changed:
+    To inspect which output files have changes, run 'snakemake --list-input-changes'.
+    To trigger a re-run, use 'snakemake -R $(snakemake --list-input-changes)'.
+Complete log: .snakemake/log/2022-07-22T085856.769525.snakemake.log
+[Atlas] CRITICAL: Command 'snakemake --snakefile /home/stamps2022/.conda/envs/atlas/lib/python3.8/site-packages/atlas/workflow/Snakefile --directory /home/stamps2022/atlas_workflow --jobs 8 --rerun-incomplete --configfile '/home/stamps2022/atlas_workflow/config.yaml' --nolock   --use-conda --conda-prefix /home/stamps2022/atlas_workflow/databases/conda_envs    --resources mem=27 mem_mb=28585 java_mem=23   --scheduler greedy  binning   ' returned non-zero exit status 1.
+```
+
+We see from the top of this message that snakemake is encountering an error.
+Let's trouble shoot this error and figure out what happened.
+
+In the error message, we see `&> SRR8859675/logs/binning/DASTool/checkm.log`. 
+This is where the log message for this command is written.
+Let's look at this error message and try and figure out what happened.
+
+
+**Challenge 4**
+Look at the stderror log message that was saved.
```

Comment:
```suggestion
Look at the log message that was saved to stderr (the standard error output location on UNIX).
```

Reply:
omg thank you this phrasing is so much better!